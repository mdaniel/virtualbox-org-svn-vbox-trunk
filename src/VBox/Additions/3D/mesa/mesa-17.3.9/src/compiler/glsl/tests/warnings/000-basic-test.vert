#version 130

void main()
{
  float foo;
  float undefined;

  foo = undefined;
}

