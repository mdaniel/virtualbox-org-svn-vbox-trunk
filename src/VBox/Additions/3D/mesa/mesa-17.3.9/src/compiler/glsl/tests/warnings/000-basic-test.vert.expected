0:8(8): warning: `undefined' used uninitialized
